### DISCUSSION TITLE
game doest start in vr

### DISCUSSION THREAD

**Discussion by Rubber15** [Raicuparta/nomai-vr#529] [General]
they warn you for this every time you start the game. but for me it still doesnt go to vr, just flat projected on the headset. is there anytrhing i am missing?

**Comment by doug141g** [comment]
Have you turned off SteamVR theater mode for Steam library> Outer Wilds> Properties> General ?

**Reply by Kean4711DUB** [reply]
I have the Epic games version and it won't open in VR
